### PR TITLE
Attempt to build with cice=5 variant

### PR DESCRIPTION
**Pull request by anton-seaice**
---
:rocket: The latest prerelease `access-esm1p6/pr24-1` at 2dc8a9cfe3ee04c0fc2f615bee92fd3c86e21e39 is here: https://github.com/ACCESS-NRI/ACCESS-ESM1.6/pull/24#issuecomment-2540363069 :rocket:

### CODE REVIEW

**Comment by anton-seaice** [comment]
Hi @harshula 

Specifying 

```
spack:
  specs:
    - access-esm1p6@git.dev_2024.12.a cice=5
```

Doesn't solve with cice5:

```
 ==> [2024-12-13-12:58:11.041841] Concretized access-esm1p6
 -   xnret6u  access-esm1p6@latest%intel@19.0.3.199+generic-tracers build_system=bundle cice=4 um=access-esm1.6 arch=linux-rocky8-x86_64_v4
 - ```

https://github.com/ACCESS-NRI/ACCESS-ESM1.6/actions/runs/12308106599/job/34352943497?pr=24#step:9:677

**Comment by anton-seaice** [comment]
> This `access-esm1.6` model will be deployed as:
> 
>     * `dev_2024.12.a cice=5` as a Release (when merged).
> 
>     * ``

Hi @CodeGat 

The bot doesn't like variants in the spack spec line of a spack.yaml for a deployment 😑

**Comment by CodeGat** [comment]
Hey @anton-seaice - noted in [this issue](https://github.com/ACCESS-NRI/build-cd/issues/185) - I'll get onto it. Does the build itself work? As in, the one with `module load access-esm1p6/pr24-1`? It could just be an issue with the comment.

**Comment by harshula** [comment]
It appears access-generic-tracers@git.dev_2024.12.0=2024.12.0 (kjyflqi) built with intel@19.0.5.281 contains wombatlite-sinking:

$ strings /[...]/0.22/release/linux-rocky8-x86_64/intel-19.0.5.281/access-generic-tracers-git.dev_2024.12.0_2024.12.0-kjyflqiymfteq4g6btxjxcpn5fra6zlo/lib64/libgtracers.a | grep -i Ecosystem
Error: Ecosystem model is not conserving nitrogen
Error: Ecosystem model is not conserving carbon

However access-generic-tracers@git.dev_2024.12.0=2024.12.0 (wcjamec) built with intel@19.0.3.199 does not:

$ strings /[...]/0.22/restricted/ukmo/release/linux-rocky8-x86_64_v4/intel-19.0.3.199/access-generic-tracers-git.dev_2024.12.0_2024.12.0-wcjamecicluyvdznrrdorcozgq3dysas/lib64/libgtracers.a  | grep -i Ecosystem

To avoid confusion, I'm going to attempt to uninstall it via the undeployment step when a PR is closed.

File: spack.yaml [hunk 1]
```diff
@@ -19,9 +19,9 @@ spack:
       require:
         - '@git.dev_2024.08.14=access-esm1.6'
         - '+access-gtracers'
-    cice4:
+    cice5:
       require:
-        - '@git.2024.05.21=access-esm1.5'
+        - '@git.access-esm1.6=access-esm1.6'
```
Comment (harshula) [inline spack.yaml:24]: The cice5 SPR doesn't contain version `access-esm1.6` yet.

Comment (anton-seaice) [reply spack.yaml:24]: Yeah agreed. But that should cause the build to fail, rather than what happened here, which spack solved and installed cice4.

Comment (harshula) [reply spack.yaml:24]: Good point! We need to work how how that happened.

Comment (harshula) [reply spack.yaml:24]: Good point! We need to work out how that happened.

**Pull request closed without merging**